docs/windows.md

```diff
@@ -162,6 +162,32 @@ $ poetry install
 $ poetry run gaphor
 ```
 
+### Debugging using Visual Studio Code
+
+Start a new PowerShell terminal, and set current directory to the project folder:
+```PowerShell
+cd <your-git-location>
```

Comment:
We don't use this term `<your-git-location>` anywhere else above. Could we please make this more consistent?

Reply:
Done, please review.